src/main/java/org/drip/model/DripUser.java

```diff
@@ -16,6 +16,8 @@
 	private String lastName;
 	@Column(name="business_name")
 	private String businessName;
+	@Column(name="account_number")
+	private int accountNumber;
```

Comment:
Given that one user can have several accounts, maybe we should move this elsewhere. Maybe have an `AccountNumber` class with `id`